tests_zeeguu_api/test_session.py

```diff
@@ -18,6 +21,87 @@ def test_create_user(self):
         rv = self.api_post('/add_user/i@i.la',form_data)
         assert len(rv.data) > 1
 
+    def test_create_user_returns_400(self):
```

Comment:
test_create_user_returns_400_if_password_not_present

this should be the name, right?